.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--
+Please help us process GitHub Issues faster by providing the following information.
+
+Note: If you have a question, please post it on the Okta Developer Forum (https://devforum.okta.com) instead. Issues in this repository are reserved for bug reports and feature requests.
+-->
+
+## I'm submitting this issue for the package(s):
+
+- [ ] jwt-verifier
+- [ ] okta-angular
+- [ ] oidc-middleware
+- [ ] okta-react
+- [ ] okta-react-native
+- [ ] okta-react-native
+- [ ] okta-vue
+
+## I'm submitting a:
+
+- [ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
+- [ ] Feature request
+- [ ] Other (Describe below)
+
+## Current behavior
+<!-- Describe how the issue manifests. -->
+
+
+## Expected behavior
+<!-- Describe what the desired behavior would be. -->
+
+
+## Minimal reproduction of the problem with instructions
+<!--
+For bug reports please provide the *STEPS TO REPRODUCE* and if possible a *MINIMAL DEMO* of the problem.
+You could use one of our sample applications to build a reproduction demo: https://github.com/okta?q=samples-js
+-->
+
+## Extra information about the use case/user story you are trying to implement
+<!-- Describe the motivation or the concrete use case. -->
+
+
+## Environment
+
+- <!-- Package Name --> Version: X.Y.Z
+- Browser:
+- OS:
+- Node version: X.Y.Z  <!-- run `node --version` -->
```

Comment:
IIRC inline comments do not work with markdown. I'd suggest:

- Package Version:
- Browser:
- OS:
- Node Version (`node -v`):
- Other:

Also, `npm version` may not be relevant much longer, so lets remove it for now.